### DISCUSSION TITLE
Selectize-powered (multi-)select fields

### DISCUSSION THREAD

**Discussion by brandonkelly** [craftcms/cms#8403] [Ideas] [answered]
We already have Selectize loaded in the CP. Maybe we should use it to spruce up custom Select and Multi-Select fields.

**Comment by jlcharette** [comment]
Yes please! This was available as a plugin in Craft 2. Make multi-select and select great again!

**Comment by bencroker** [comment]
It's 2023 and Selectize is _still_ being used in the CP, and Select and Multi-Select fields could still _really_ benefit from it!

**Reply by brandonkelly** [reply]
Thanks for the reminder!

**Comment by brandonkelly** [comment, marked as answer]
Just made this change for Craft 4.4. (#12608)

<img width="637" alt="Dropdown and Multi-select fields, powered by Selectize." src="https://user-images.githubusercontent.com/47792/216187133-d0e50240-f8e5-4294-86bc-e22f156911c4.png">

**Reply by bencroker** [reply]
Wonderful, thanks!!

**Reply by brandonkelly** [reply]
4.4 is out!
